### PR TITLE
[Feature] Setup Travis CI to run unit tests for the OntologyQueryInterface

### LINKED ISSUE #20: Setup CI and add unit tests for the ontology query interface
This would require:
* Setting up travis (the travis file in the [`black-box-tools`](https://github.com/ropod-project/black-box-tools/blob/master/.travis.yml) repository could be used as a sample)
* Adding a test ontology (the simple ontology from the [knowledge base tutorial](https://github.com/b-it-bots/mas_tutorials/blob/master/domestic_robotics/knowledge_base.ipynb) could be used for that purpose)
* Adding a unit test script for the ontology interface

### PR DESCRIPTION
**Pull request by Sushant-Chavan**
This PR implements unit-tests for validating the `OntologyQueryInterface` using a new, simple apartment ontology. These unit-tests can then be used during CI with Travis. 

## Changelog

* Added a new bare minimal apartment ontology called `sample.owl` based on the [tutorials](https://github.com/b-it-bots/mas_tutorials/blob/master/domestic_robotics/knowledge_base.ipynb)
* Added unit tests for all the query APIs of the `OntologyQueryInterface`
* Fixed the issues detected by the unit tests in the `OntologyQueryInterface`
* Added TravisCI config file to install ROS and run the unit-tests

## Related PRs

Closes #20   
Related https://github.com/b-it-bots/mas_tutorials/issues/14

## Checklist:

- [x] My code doesn't contain unnecessary comment blocks (e.g. unused code, templates of `package.xml` or `CMakeLists.txt`)
- [x] I have updated the `package.xml` and `CMakeLists.txt` with the correct dependencies.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Sushant-Chavan** [comment]
@alex-mitrevski I need your help for installing the mas_knowledge_base repository in Travis. The `mas_knowledge_utils` module seems to be installed according to the [logs](https://travis-ci.com/github/b-it-bots/mas_knowledge_base/builds/153350858). But when running the test script, python complains that the `mas_knowledge_utils` module is not found. Any ideas on how I could fix this?

**Comment by alex-mitrevski** [comment]
Hmm, not sure what the problem could be. Maybe try installing the package without `--user`?

**Comment by Sushant-Chavan** [comment]
@alex-mitrevski, I already tried that, but that doesn't work either. Can I go ahead and install ROS kinetic bare bones in Travis? Although that seems a lot of overhead to me, I can't think of any other solution due to my lack of experience with distutils and setting up custom python modules.

**Comment by alex-mitrevski** [comment]
Sure, go ahead. We'll anyway need that for the knowledge base interface.

File: common/tests/ontology_query_interface_unit_tests.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    @author Sushant Chavan
+    @contact sushant.chavan@smail.inf.h-brs.de
+
+    '''
+
+    def setUp(self):
+        # Get directory paths
+        script_dir = os.path.abspath(os.path.dirname(__file__))
+        ontology_dir = os.path.join(os.path.dirname(script_dir), "ontology")
+        assertions_dir = os.path.join(os.path.dirname(script_dir), "assertions")
```
Comment (alex-mitrevski) [inline common/tests/ontology_query_interface_unit_tests.py]: This variable is not used anywhere; can you please remove it?

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]

File: common/tests/ontology_query_interface_unit_tests.py (hunk 1 above)
Comment (Sushant-Chavan) [reply common/tests/ontology_query_interface_unit_tests.py]: Removed the variable in the new commit.

**Review by alex-mitrevski — APPROVED** [review]

**Pull request merged**